src/app/tools/weak-passwords-report.component.ts

```diff
@@ -63,9 +66,31 @@ export class WeakPasswordsReportComponent extends CipherReportComponent implemen
                 this.passwordStrengthCache.set(cacheKey, result.score);
             }
             const score = this.passwordStrengthCache.get(cacheKey);
-            if (score != null && score <= 2) {
-                this.passwordStrengthMap.set(c.id, this.scoreKey(score));
+            if (score != null && (score <= 2 || this.showAllCheckbox)) {
+
+                let charSetSize = 0;
+                if (c.login.password.match(/[abcdefghijklmnopqrstuvwxyz]/g) != null) {
```

Comment:
If the password contains any lowercase letter, add 26, as there are 26 possible lowercase letters.